### ISSUE TITLE
How to make modifications?

### ISSUE THREAD

**Issue by GLivshits** [microsoft/MMdnn#927]
Hello. I want to convert mxnet model to pytorch. I've found out bad rename mappings in mxnet_parser.py, which is easily solved. However, it's not obvious how to make new rename functions to nodes with attributes. For example, BatchNorm mx attributes mapped to some different names, and it is not obvious at all how to map arguments for new ops. Where the mapping can be found?
